02-getting-started/projects/final/iosApp/iosApp/ContentView.swift

```diff
@@ -2,15 +2,14 @@ import SwiftUI
 import shared
 
 struct ContentView: View {
-	let greet = Greeting().greeting()
 
 	var body: some View {
-		Text(greet)
+		Text("Hello")
```

Comment:
Isn't the final app supposed to have the compilation issue here?
The reader is supposed to find this issue and fix the compilation as part of the challenge

Reply:
No. The challenge is supposed to have the original. That is what the user is supposed to fix.

Reply:
From my understanding, the challenge project in the materials is supposed to be the solution of the challenge. The final project will be the base on top of which the reader will build the solution

Example: https://www.raywenderlich.com/books/ios-animations-by-tutorials/v6.0/chapters/6-view-animations-in-practice#toc-chapter-010-anchor-005
The challenge asks to animate a view
The final project doesn't contain the animation: https://github.com/raywenderlich/iat-materials/blob/editions/6.0/06-view-animations-in-practice/projects/final/Flight%20Info/ViewController.swift
The challenge project contains the solution to the challenge: https://github.com/raywenderlich/iat-materials/blob/editions/6.0/06-view-animations-in-practice/projects/challenge/Flight%20Info/ViewController.swift#L97

Reply:
Hmm. I guess you are right. But the final project should also have the solution as well. It don't want it to be broken

Reply:
Let's keep the solution at both places then: final and challenge

Reply:
@kevindmoore Can you please resolve this comment sot that we can send this chapter to pagesetting